man/systemd-bsod.xml

```diff
@@ -43,7 +43,16 @@
     <variablelist>
       <xi:include href="standard-options.xml" xpointer="help" />
       <xi:include href="standard-options.xml" xpointer="version" />
+
+     <varlistentry>
+        <term><option>-w</option></term>
+        <term><option>-wait</option></term>
```

Comment:
Hmm, I think this should be called "continuous" or so, since I think sooner or late rwe should probably teach the thing to cycle through all matching messages, i.e. display the first emergency message for 5s, then switch to the next for 5s, and so on, eventually beginning from the top again.

To me "--wait" suggests we just wait for one event, and then are done. But I think we should just keep running here as soon as the first matching message is found. Initially we'd just show the first, but the cycling should be added too eventually.

Reply:
also long options are prefixed with two dashes, not just one, i.e. `--continuous`

Reply:
or `--continously`, @bluca, what do you say?

Reply:
> Hmm, I think this should be called "continuous" or so, since I think sooner or late rwe should probably teach the thing to cycle through all matching messages, i.e. display the first emergency message for 5s, then switch to the next for 5s, and so on, eventually beginning from the top again.
> 
> To me "--wait" suggests we just wait for one event, and then are done. But I think we should just keep running here as soon as the first matching message is found. Initially we'd just show the first, but the cycling should be added too eventually.

Awesome.
On it!

Reply:
> Hmm, I think this should be called "continuous" or so, since I think sooner or late rwe should probably teach the thing to cycle through all matching messages, i.e. display the first emergency message for 5s, then switch to the next for 5s, and so on, eventually beginning from the top again.
> 
> To me "--wait" suggests we just wait for one event, and then are done. But I think we should just keep running here as soon as the first matching message is found. Initially we'd just show the first, but the cycling should be added too eventually.

Please let me be part of the "we" when it's time to teach this thing to cycle through the messages in intervals of 5s sir.